src/REGothEngine.cpp

```diff
@@ -181,12 +222,11 @@ int ::REGoth::main(REGothEngine& regoth, int argc, char** argv)
 
   regoth.loadCachedResourceManifests();
 
+  regoth.setShaders();
   regoth.setupInput();
   regoth.setupMainCamera();
   regoth.setupScene();
 
-  regoth.saveCachedResourceManifests();
```

Comment:
You forgot to remove the same line in `src/main.cpp`, so the `REGoth` target doesn't compile.